chain/jsonrpc/res/debug.html

```diff
@@ -24,9 +24,16 @@
                 type: "GET",
                 url: "/status",
                 success: data => {
+                    let binaryText = $('<span>')
+                        .text(`version "${data.version.version}" built with rustc "${data.version.rustc_version}" at `);
+                    binaryText.append(
+                        $('<a>')
+                            .text(data.version.build)
+                            .attr('href', `https://github.com/near/nearcore/tree/${data.version.build}`));
```

Comment:
Since we’re interpreting the values, perhaps don’t include version if it’s `trunk`?

Reply:
Explicitly saying it's built as "trunk" may also be helpful? (Not exactly sure how these are determined)

Reply:
Version comes from version in neard/Cargo.toml file. It’s only set on release builds.  Build comes from git describe.  The strings are constructed in neard/build.rs.  To be more descriptive the format could be:
- ‘Release {version}’ if version == build with version being the link,
- ‘Release {version} (build {build})’ if version is not ‘trunk’ or
- ‘Nightly build {build}’ if version is ‘trunk’.

But up to you.

Reply:
I really like that :) Thanks!